src/Microsoft.AspNet.Mvc.Localization/Internal/MvcLocalizationServices.cs

```diff
@@ -15,6 +17,14 @@ public static class MvcLocalizationServices
             IServiceCollection services,
             LanguageViewLocationExpanderFormat format)
         {
+            AddLocalizationServices(services, format, setupAction: null);
+        }
```

Comment:
Tests call it. But I will change the tests to use the other method and will remove this one.